AsyncDisplayKit/Private/ASInternalHelpers.m

```diff
@@ -107,7 +107,7 @@ BOOL ASClassRequiresMainThreadDeallocation(Class class)
 Class _Nullable ASGetClassFromType(const char *type)
 {
   // Class types all start with @"
-  if (strncmp(type, "@\"", 2) != 0) {
+  if (type == nil || strncmp(type, "@\"", 2) != 0) {
```

Comment:
It's super minor point, but we should use `NULL` instead of `nil` here since it's not an object. 🤓